ios/gutenberg/GutenbergViewController.swift

```diff
@@ -113,6 +113,26 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
         alertController.popoverPresentationController?.permittedArrowDirections = .any
         present(alertController, animated: true, completion: nil)
     }
+
+    func debug(message: String) {
```

Comment:
Memo: we'll inject actual logger in https://github.com/wordpress-mobile/WordPress-iOS/blob/14ba744f99517971c3a2afe1297ba6da9866616d/WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift#L315 on WP-iOS side